qemu/tests/cfg/ansible_test.cfg

```diff
@@ -15,3 +15,16 @@
             type = ansible_with_responsive_migration
             wait_response_timeout = 1800
             #mq_listen_port =
+        - nested_block_resize:
```

Comment:
Please remove this part in a single cfg file, ansible_test is not only used for nested testing.

Reply:
Updated